R/data.table.R

```diff
@@ -1015,7 +1015,7 @@ replace_dot_alias = function(e) {
           } else negate_sdcols = FALSE
           # fix for #1216, make sure the parentheses are peeled from expr of the form (((1:4)))
           while(colsub %iscall% "(") colsub = as.list(colsub)[[-1L]]
-          if (colsub %iscall% ':' && length(colsub)==3L && !is.call(colsub[[2L]]) && !is.call(colsub[[3]])) {
+          if (colsub %iscall% ':' && length(colsub)==3L && !is.call(colsub[[2L]]) && !is.call(colsub[[3L]])) {
```

Comment:
Changes in this file seem like not fitting in this PR

Reply:
I agree, but they were in the other one...

Reply:
These were minimal and were initiated based on the release checks. Maybe not a perfect fit but it does line up master with the minor changes in the CRAN version.